content/docs-v1/collection-catalogues.md

```diff
@@ -270,15 +273,31 @@ object_list:
       - id: 
   ```   
 
-Then, in the object filter, use the `field`/`label` attribute to ensure the data is being filtered by `sort_year` but displays as `year`. You can use this for other attributes such as `sort_location`, etc. 
+  ``` yaml
+object_list:
+  - id: 
+    artist:
+    title: 
+    year: "1993"
+    sort_year: "1993"
```

Comment:
I used this as the example because I want to make it clear that even if your year is written normally, you will still need to include `sort_year`. If that attribute is used once then it needs to be used everywhere. Otherwise, you will have two filterable categories, `year` and `sort_year`.